tests/run-make/issue-11908/rmake.rs

```diff
@@ -0,0 +1,35 @@
+// A path which contains the same rlib or dylib in two locations
+// should not cause an assertion panic in the compiler.
+// This test tries to replicate the linked issue and checks
+// if the bugged error makes a resurgence.
+
+// Note that this relies on `liburl` to be in the path somewhere else.
+// Our aux-built libraries will collide with liburl (they have
+// the same version listed)
```

Comment:
wtf? This seems rather fragile?

Reply:
Is this comment even accurate anymore?

Reply:
Tentatively, it seems like the test author, [in this commit](https://github.com/rust-lang/rust/commit/15b680ae86eefae754c3a348d89207fa100b4ca6), implemented the directory moving commands to fix the collision, but didn't remove the comment. I have removed it, but let me know if you think it should be kept in some way, since we are not 100% sure.

Reply:
Unless this fails because of liburl, the comment seems outdated